c2corg_ui/format/ltag.py

```diff
@@ -0,0 +1,64 @@
+"""
+LTag Extension for Python-Markdown
+====================================
+
+See https://regex101.com/r/g2Zjqn/2/ for regex tests
+
+"""
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from markdown.extensions import Extension
+from markdown.blockprocessors import BlockProcessor
+from markdown.util import etree
+import re
+
+
+class LTagProcessor(BlockProcessor):
+    """ Process Tables. """
+
+    RE_LTAG_BLOCK = re.compile(r'^(?:L#(?:[^|\n]*(?:\|[^|\n]*)+)\n?)+$')
+
+    def __init__(self, parser):
+        super(LTagProcessor, self).__init__(parser)
+
+    def test(self, parent, block):
+        is_table = self.RE_LTAG_BLOCK.search(block) is not None
+
+        return is_table
```

Comment:
Perhaps you could use a more explicit name for this function and simplify a bit its code, like this
```
def is_table(self, parent, block):
￼    return self.RE_LTAG_BLOCK.search(block) is not None
```
?